app/components/containers/HTTPError.js

```diff
@@ -1,5 +1,11 @@
+/* eslint-disable global-require */
 import React from 'react';
 
-const HTTPError = ({ code }) => (<h1>Error {code}</h1>);
+const HTTPError = ({ code }) => (
+  <div>
```

Comment:
avoid using `div` if possible. for this case, it might make more sense to use `main`